guides/source/action_view_overview.md

```diff
@@ -613,22 +831,20 @@ Guide](form_helpers.html).
 Localized Views
 ---------------
 
-Action View has the ability to render different templates depending on the current locale.
-
-For example, suppose you have an `ArticlesController` with a show action. By default, calling this action will render `app/views/articles/show.html.erb`. But if you set `I18n.locale = :de`, then `app/views/articles/show.de.html.erb` will be rendered instead. If the localized template isn't present, the undecorated version will be used. This means you're not required to provide localized views for all cases, but they will be preferred and used if available.
-
-You can use the same technique to localize the rescue files in your public directory. For example, setting `I18n.locale = :de` and creating `public/500.de.html` and `public/404.de.html` would allow you to have localized rescue pages.
-
-Since Rails doesn't restrict the symbols that you use to set I18n.locale, you can leverage this system to display different content depending on anything you like. For example, suppose you have some "expert" users that should see different pages from "normal" users. You could add the following to `app/controllers/application_controller.rb`:
+Action View has the ability to render different templates depending on the
+current locale.
 
-```ruby
-before_action :set_expert_locale
-
-def set_expert_locale
-  I18n.locale = :expert if current_user.expert?
-end
-```
+For example, suppose you have an `ArticlesController` with a show action. By
+default, calling this action will render `app/views/articles/show.html.erb`. But
+if you set `I18n.locale = :de`, then `app/views/articles/show.de.html.erb` will
+be rendered instead. If the localized template isn't present, the undecorated
+version will be used. This means you're not required to provide localized views
+for all cases, but they will be preferred and used if available.
```

Comment:
This made me pause for a moment, when we say "then .../show.de.html.erb will be rendered instead". We go on to explain what happens if the localized version is not present, but that first portion is the one that made me "wait".

Maybe we can word it to try to indicate it will "try" that localized first? Here's some food for thought suggestion, feel free to take it or drop it:

```suggestion
if you set `I18n.locale = :de`, then Action View will try to render the template
`app/views/articles/show.de.html.erb` first. If the localized template isn't
present, the undecorated version will be used. This means you're not required to
provide localized views for all cases, but they will be preferred and used if
available.
```